From bb9a6ddde640110e071c2e54fb4186fd25d887ca Mon Sep 17 00:00:00 2001
From: Abe Lopez <abe.lopez@onsemi.com>
Date: Tue, 7 Nov 2017 13:52:55 -0800
Subject: [PATCH] Correct fleece call of nullValue

---
 Fleece/Value.hh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Fleece/Value.hh b/Fleece/Value.hh
index 7c96775..7e0036c 100644
--- a/Fleece/Value.hh
+++ b/Fleece/Value.hh
@@ -48,7 +48,7 @@ namespace fleece {
     class Null {
     };
 
-    constexpr Null nullValue;
+    constexpr Null nullValue();
 
 
     /* An encoded data value */
-- 
2.7.4

