diff --git a/CMakeLists.txt b/CMakeLists.txt
index a14e59e5..970f3bbb 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -698,22 +698,22 @@ endif ()

 if (MSVC)
     # default for all sources is to use precompiled headers
-    foreach(source ${sources})
-        set_source_files_properties(${source}
-            PROPERTIES
-            COMPILE_FLAGS "/Yuprecompiled.hpp"
-            )
-    endforeach()
+#    foreach(source ${sources})
+#        set_source_files_properties(${source}
+#            PROPERTIES
+#            COMPILE_FLAGS "/Yuprecompiled.hpp"
+#            )
+#    endforeach()
     # create precompiled header
-    set_source_files_properties(${CMAKE_CURRENT_SOURCE_DIR}/src/precompiled.cpp
-        PROPERTIES
-        COMPILE_FLAGS "/Ycprecompiled.hpp"
-        )
+#    set_source_files_properties(${CMAKE_CURRENT_SOURCE_DIR}/src/precompiled.cpp
+#        PROPERTIES
+#        COMPILE_FLAGS "/Ycprecompiled.hpp"
+#        )
     # C and C++ can not use the same precompiled header
-    set_source_files_properties(${CMAKE_CURRENT_SOURCE_DIR}/src/tweetnacl.c
-        PROPERTIES
-        COMPILE_FLAGS "/Y-"
-        )
+#    set_source_files_properties(${CMAKE_CURRENT_SOURCE_DIR}/src/tweetnacl.c
+#        PROPERTIES
+#        COMPILE_FLAGS "/Y-"
+#        )
 endif ()


