diff --git a/web/app.js b/web/app.js
index a4229a81..4304e24a 100644
--- a/web/app.js
+++ b/web/app.js
@@ -1415,7 +1415,7 @@ let PDFViewerApplication = {
 let validateFileURL;
 if (typeof PDFJSDev === 'undefined' || PDFJSDev.test('GENERIC')) {
   const HOSTED_VIEWER_ORIGINS = ['null',
-    'http://mozilla.github.io', 'https://mozilla.github.io'];
+    'http://mozilla.github.io', 'https://mozilla.github.io', 'file://', 'qrc://'];
   validateFileURL = function validateFileURL(file) {
     if (file === undefined) {
       return;
-- 
2.20.1

