diff --git a/vendor/couchbase-lite-core/Networking/WebSockets/WebSocketImpl.cc b/vendor/couchbase-lite-core/Networking/WebSockets/WebSocketImpl.cc
index 7f88f8d6..50140c18 100644
--- a/vendor/couchbase-lite-core/Networking/WebSockets/WebSocketImpl.cc
+++ b/vendor/couchbase-lite-core/Networking/WebSockets/WebSocketImpl.cc
@@ -37,7 +37,7 @@ namespace litecore { namespace websocket {
     constexpr long WebSocketImpl::kConnectTimeoutSecs;

     // Default interval at which to send PING messages (configurable via options)
-    static constexpr auto kDefaultHeartbeatInterval = chrono::seconds(5 * 60);
+    static constexpr auto kDefaultHeartbeatInterval = chrono::seconds(30);

     // Timeout for disconnecting if no PONG response received
     static constexpr auto kPongTimeout  = chrono::seconds(10);
diff --git a/vendor/couchbase-lite-core/Crypto/PublicKey.hh b/vendor/couchbase-lite-core/Crypto/PublicKey.hh
index de513642..8909d8fc 100644
--- a/vendor/couchbase-lite-core/Crypto/PublicKey.hh
+++ b/vendor/couchbase-lite-core/Crypto/PublicKey.hh
@@ -147,9 +147,6 @@ namespace litecore { namespace crypto {
     };


-#ifdef __APPLE__                // TODO: Implement subclasses for other platforms
-    #define PERSISTENT_PRIVATE_KEY_AVAILABLE
-#endif

 #ifdef PERSISTENT_PRIVATE_KEY_AVAILABLE

diff --git a/vendor/couchbase-lite-core/Replicator/c4RemoteReplicator.hh b/vendor/couchbase-lite-core/Replicator/c4RemoteReplicator.hh
index a4e51654..407aeffb 100644
--- a/vendor/couchbase-lite-core/Replicator/c4RemoteReplicator.hh
+++ b/vendor/couchbase-lite-core/Replicator/c4RemoteReplicator.hh
@@ -27,7 +27,7 @@ namespace c4Internal {

         // Default maximum number of retry attempts before replications give up.
         // These can be overridden by setting the option `kC4ReplicatorOptionMaxRetries`.
-        static constexpr unsigned kMaxOneShotRetryCount = 2;
+        static constexpr unsigned kMaxOneShotRetryCount = 0;
         static constexpr unsigned kMaxContinuousRetryCount = UINT_MAX;

         // Longest possible retry delay, in seconds. The delay doubles on each failed retry
